### DISCUSSION TITLE
Unable to update composer/ Laravel. Getting the erro

### DISCUSSION THREAD

**Discussion by ghost** [spatie/laravel-medialibrary#2010] [Q&A] [answered]
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package laravel/laravel No version set (parsed as 1.0.0) is satisfiable by laravel/laravel[No version set (parsed as 1.0.0)] but these conflict with your requirements or minimum-stability.

**Comment by freekmurze** [comment, marked as answer]
Seems like this problem is not related to the package.

**Reply by ghost** [reply]
I am also clueless how to solve this.
